### DISCUSSION TITLE
Do you want to install @nuxt/image package?

### DISCUSSION THREAD

**Discussion by dbzx10299** [nuxt/nuxt#27978] [General]
I am on the latest version of nuxt `3.12.3` and every time I run `npm run dev` I get this prompt in the terminal
```
Do you want to install @nuxt/image package?
Yes / no
```

I don't need this package and if I select `no`, I have to keep selecting no each time I use the dev command.

The prompt keeps appearing now as well. I am not sure if this is a bug, or if there is a way to turn this off?

```
Do you want to update your nuxt.config to set compatibilityDate: '2024-07-02'?
Yes / no
```

**Comment by 7Koston** [comment]
The cause of @nuxt/image prompt is described here:
https://github.com/nuxt/nuxt/issues/27854 -> https://github.com/nuxt-modules/mdc/issues/234

**Comment by timothymarois** [comment]
Also on this. not good. Its actually crashing my npm dev builds. You can not select no. No other project is doing this aside from this one. never used nuxt img in any project. 

<img width="520" alt="Screenshot 2024-07-06 at 12 04 02 PM" src="https://github.com/nuxt/nuxt/assets/2489333/a399e580-501c-49fc-9bab-d50a0684fef6">

**Comment by 7Koston** [comment]
Workaround is just to install `@nuxt/image` and wait for issues above to get resolved.
Its gonna be tree shaked out anyways if you are not using `NuxtImage` or `NuxtPicture` components.

Or you can submit pull request with the fix to `nuxt-mdc` :D

**Comment by Aareksio** [comment]
`compatibilityDate` should be a separate topic, after selecting "yes" once it should update your nuxt.config.ts with a new value and never ask again. It is not yet documented anywhere, but the vague idea is to have it pin you to certain underlying behavior you may need, while allowing _other_ framework components to get updates.

**Comment by nort3x** [comment]
this is so stupid 😄
surely you are to busy to find and install that manually yet you have time to search and find this issue, sallow a hack as solution and wait for a fix for a feature you can't opt-out from,
these are just annoying counter intuitive features
